e2e/cleanup_test.go

```diff
@@ -276,4 +274,14 @@ spec:
 		stdout, stderr, err := kubectl("delete", "ns", cleanupTest)
 		Expect(err).ShouldNot(HaveOccurred(), "stdout=%s, stderr=%s", stdout, stderr)
 	})
+
+	It("cleanup", func() {
```

Comment:
```suggestion
	It("should cleanup volumes", func() {
```

Reply:
@llamerada-jp
This is a not test, but cleanup logic.
I think using `Should` is confusing.

Reply:
Since this is a cleanup process, we should use `AfterEach` rather than `It`.
However, if we use `AfterEach`, the cleanup will be run by each `It`.
So, we use `It` for cleanup, although it is a strange way to use `It`.

Reply:
I discussed this with my teams, we concluded to use the same message as the others to avoid confusion.